ragna/deploy/_ui/central_view.py

```diff
@@ -613,26 +613,27 @@ def header(self):
 
                 chat_documents_pills.append(pill)
 
-        chat_info_button = None
-        if self.current_chat is not None:
-            button_name = f"{self.current_chat['metadata']['assistant']} | {self.current_chat['metadata']['source_storage']}"
-            chat_info_button = pn.widgets.Button(
+        button_name = f"{self.current_chat['metadata']['assistant']} | {self.current_chat['metadata']['source_storage']}"
+        if hasattr(self, "chat_info_button"):
+            self.chat_info_button.name = button_name
+        else:
+            self.chat_info_button = pn.widgets.Button(
                 name=button_name, button_style="outline", icon="info-circle"
             )
-            chat_info_button.stylesheets.append(
-                """
-                                        :host {  
-                                                    margin-top:10px;
-                                        }
-                                        
-                                    """
-            )
-            chat_info_button.on_click(self.on_click_chat_info_wrapper)
+        self.chat_info_button.stylesheets.append(
+            """
+                                    :host {  
+                                                margin-top:10px;
+                                    }
+                                    
+                                """
+        )
+        self.chat_info_button.on_click(self.on_click_chat_info_wrapper)
```

Comment:
1. Unless I'm missing something here, these could be static too. Meaning, we can put that in the constructor of the class, since neither the CSS nor the callback is dependent on anything in this method.
2. Since we are already touching this: could you please format this absolutely hideous CSS formatting?